fe/src/main/java/org/apache/doris/catalog/OlapTable.java

```diff
@@ -1011,17 +1010,17 @@ public void readFields(DataInput in) throws IOException {
                 tableProperty = TableProperty.read(in);
             }
         }
+        // inMemory
+        if (Catalog.getCurrentCatalogJournalVersion() >= FeMetaVersion.VERSION_72) {
+            isInMemory = in.readBoolean();
```

Comment:
Could we put this `isInMemory` in `tableProperty` to avoid modifying meta version?

Reply:
OK.
I didn't notice the `TableProperty`,  Thanks for your reminder.